src/osgPlugins/ogr/ReaderWriterOGR.cpp

```diff
@@ -262,7 +262,7 @@ class ReaderWriterOGR : public osgDB::ReaderWriter
 
     osg::Geometry* multiPointToDrawable(OGRMultiPoint* mpoint) const
     {
-        osg::Geometry* geom = new osg::Geometry;
+        osg::ref_ptr<osg::Geometry> geom = new osg::Geometry;
```

Comment:
is geom properly used?, seems to me it's only initialized and then used below only for reading. I think it's not required

Reply:
Yes also agree, fixed. Thanks @rafagaitan